Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -3601,9 +3601,21 @@ script:
       name: reporttype
     - description: The report name.
       name: reportname
-    - defaultValue: keygen,config,commit,op,report,log,import,export,user-id,version
```

Comment:
double check that this isnt a breaking change

Reply:
checked it out and this isn't a breaking change as you can only pass in 1 type per api request and not a list